vision_msgs_rviz_plugins/package.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>vision_msgs_rviz_plugins</name>
+  <version>0.1.0</version>
+  <description>RVIZ2 plugins for visualizing vision_msgs</description>
+  <maintainer email="georg.novotny@aon.at">Georg Novotny</maintainer>
+  <maintainer email="stevenmacenski@gmail.com">Steve Macenski</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_python</buildtool_depend>
+
+
+  <depend>yaml-cpp</depend>
```

Comment:
I don't think this is a rosdep key, I think there's a yaml vendor

Reply:
These are important because when I try to release this as a binary, the build farm will fail to find the dependency and fail the job. The package.xml having all the dependencies + be properly named is critical